### PR TITLE
fix: floating ui elements set max-content for inline-size

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #10731

## Summary

- Follows [`floating-ui`'s initial layout recommendation](https://floating-ui.com/docs/computeposition#initial-layout).
- Combobox wasn't correctly handling the size when in fixed overlay positioning
  - Example: https://codepen.io/Matt-Driscoll/pen/raByNmQ?editors=1010
- updated tooltip and popover stories where stories were being cut off due to sizing.

BEGIN_COMMIT_OVERRIDE
END_COMMIT_OVERRIDE

### CODE REVIEW

File: packages/calcite-components/src/components/popover/popover.stories.ts [hunk 1]
```diff
@@ -124,18 +124,20 @@ export const flipPlacements_TestOnly = (): string => html`
...
-      heading="Dreams didn't make us kings. Dragons did. 🐉"
-      reference-element="reference-element"
-      placement="auto"
-      open
-      closable
-      scale="m"
-    >
-      ${contentHTML}
-    </calcite-popover>
+  <div style="width: 800px; height:800px;">
```
Comment (jcfranco) [inline packages/calcite-components/src/components/popover/popover.stories.ts:127]: Sidebar: do you think adding a shared set of storybook decorators (similar to [this](https://www.chromatic.com/docs/troubleshooting-snapshots/#use-fixed-height-wrappers-for-portal-components) or [this](https://www.chromatic.com/docs/troubleshooting-snapshots/#why-isnt-my-modal-or-dialog-captured)) to normalize the dimensions of similar stories would be useful?

**Review by jcfranco — APPROVED** [review]
✨🚀✨

File: packages/calcite-components/src/components/popover/popover.stories.ts (hunk 1 above)
Comment (driskull) [reply packages/calcite-components/src/components/popover/popover.stories.ts:127]: Yes that would be very helpful. We already do variations of this for some components.

Comment (jcfranco) [reply packages/calcite-components/src/components/popover/popover.stories.ts:127]: Sweet. I'll create an issue for this.

**Pull request merged**